com.microsoft.mrtk.spatialmanipulation/BoundsControl/BoundsHandleInteractable.cs

```diff
@@ -101,9 +133,27 @@ protected virtual void LateUpdate()
                 colliders[0].enabled = !IsOccluded;
             }
 
-            // Maintain global scale of the handle(s).
+            // Maintain the aspect ratio/proportion of the handles, globally.
             transform.localScale = Vector3.one;
-            transform.localScale = new Vector3(1.0f / transform.lossyScale.x, 1.0f / transform.lossyScale.y, 1.0f / transform.lossyScale.z);
+            transform.localScale = new Vector3(1.0f / transform.lossyScale.x,
+                                               1.0f / transform.lossyScale.y,
+                                               1.0f / transform.lossyScale.z);
+            
+            // If we don't want to maintain the overall *size*, we scale
+            // by the maximum component of the box so that the handles grow/shrink
+            // with the overall box manipulation.
+            if (!maintainGlobalSize)
+            {
+                transform.localScale = transform.localScale * (MaxComponent(transform.parent.lossyScale) / initialParentScale);
+            }
+        }
+
+        private float MaxComponent(Vector3 v)
+        {
+            Vector3 abs = new Vector3(Mathf.Abs(v.x), Mathf.Abs(v.y), Mathf.Abs(v.z));
```

Comment:
Probably doesn't do much different logically, but you should be able to use https://docs.unity3d.com/ScriptReference/Mathf.Max.html here, passing in these three `Mathf.Abs` values.

Reply:
Yeah, sorry my brain was fried, not sure why I did this, lol.

Reply:
Fixed, that was really questionable of me

Reply:
@Zee2 I'm not sure I see any commits in this PR since before my comment